tests/tests_suite_test.go

```diff
@@ -247,7 +253,9 @@ func start(cmdLine string, args ...interface{}) (*Session, error) {
 func startCmd(command Cmd) (*Session, error) {
 	cmd := exec.Command("/bin/sh", "-c", command.CommandLineString)
 	cmd.Env = command.Env
-	io.WriteString(GinkgoWriter, fmt.Sprintf("$ %s\n", command.CommandLineString))
+	if debug {
```

Comment:
Oops! My bad @mboersma ... haha had thought I added the `if debug` earlier and was wondering where it went.  So, apologies for causing the first go-round of the bug (#71) and removing this change now.